src/Entity/Type.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\TypeRepository")
+ */
+class Type
```

Comment:
@daria-sieroshtan 
"type" is veeeeery bad name, way too ambiguous
Is it "LarpType"?

Reply:
@daria-sieroshtan 
OK.
Name it accordingly )

Reply:
but it is property of event, and everything here is about events. Should I name everything like eventDate, eventName and so on? I guess it will be too much

Reply:
@daria-sieroshtan 
It is ok to have event property named "type"
It is not OK to have class named "Type"